mail/mail-server/Portfile

```diff
@@ -329,6 +341,29 @@ destroot {
 
 destroot.keepdirs ${destroot}${prefix}/var/log/mail
 
+# Workaround for issue with dovecot version 2.3.17 on macOS 12
+# https://www.mail-archive.com/dovecot@dovecot.org/msg84784.html
+if { ${os.major} >= 21 } {
```

Comment:
please include the usual check for `os.platform` as well

Reply:
Thanks, done.